File: NetCore/Lab02/Model/AccountDao.cs
namespace Lab02.Model;

public interface IAccountDao
{
    Member GetMemberForLogin(string account);

    void SetLoginFailedCount(string account);

    int GetLoginFailedCount();
}

public class AccountDao : IAccountDao
{
    public virtual Member GetMemberForLogin(string account)
    {
        throw new System.NotImplementedException();
    }

    public void SetLoginFailedCount(string account)
    {
        throw new System.NotImplementedException();
    }

    public int GetLoginFailedCount()
    {
        throw new System.NotImplementedException();
    }
}

File: kotlin/lab02/src/main/kotlin/lab02/model/AccountDao.kt
package lab02.model

class AccountDao {
    fun getMemberForLogin(account: String): Member {
        throw NotImplementedError()
    }

    fun setLoginFailedCount(account: String) {
        throw NotImplementedError()
    }

    fun getLoginFailedCount(): Int {
        throw NotImplementedError()
    }
}
